newrelic/data_source_newrelic_alert_channel.go

```diff
@@ -57,10 +157,7 @@ func dataSourceNewRelicAlertChannelRead(d *schema.ResourceData, meta interface{}
 	}
 
 	d.SetId(strconv.Itoa(channel.ID))
```

Comment:
Should we include this as part of `flattenAlertChannel`? Would put all the "Set" methods in single area.

Reply:
This flatten method is shared between the alert channel data source and resource. Usage of `SetID` changes based on whether or not the resource is found on the backend, so I'm not sure we can have a common treatment for that one.  `policy_ids` isn't an attribute on the resource currently.

Reply:
Ah, okie dokie.